src/transformers/models/xlnet/configuration_xlnet.py

```diff
@@ -57,7 +57,9 @@ class XLNetConfig(PretrainedConfig):
         untie_r (`bool`, *optional*, defaults to `True`):
             Whether or not to untie relative position biases
         attn_type (`str`, *optional*, defaults to `"bi"`):
-            The attention type used by the model. Set `"bi"` for XLNet, `"uni"` for Transformer-XL.
+            The attention type used by the model. Set `"bi"` for XLNet.`"uni"` is not longer supperted. If you want to
+            use Transformer-XL, use a
+            [`Transformer-XLModel`](https://huggingface.co/docs/transformers/model_doc/transfo-xl).
```

Comment:
The link is very weird. I'd just do
```suggestion
            The attention type used by the model. Set `"bi"` for XLNet.`"uni"` is no longer supported, use [Transformer-XL](./transfo-xl) instead.
```